README.md

```diff
@@ -67,6 +67,7 @@ welcomed.
 * Heather Macbeth (@hrmacbeth): geometry, analysis
 * Patrick Massot (@patrickmassot): documentation, topology
 * Bhavik Mehta (@b-mehta): category theory, combinatorics
+* Oliver Nash (@ocfnash): algebra, geometry, topology
 * Scott Morrison (@semorrison): category theory
```

Comment:
This would be the correct alphabetical order:
```suggestion
* Scott Morrison (@semorrison): category theory
* Oliver Nash (@ocfnash): algebra, geometry, topology
```

Reply:
Thanks.

I was so excited I forgot my abc.